src/BlazorUI/Bit.BlazorUI/Components/Notifications/Persona/BitPersona.scss

```
@@ -81,15 +81,15 @@
 }
 
 .bit-prs-img {
-    top: 0;
-    left: 0;
+    // top: 0;
+    // left: 0;
     opacity: 1;
-    z-index: 1;
+    // z-index: 1;
```

Stylelint (17.12.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)